stock_move_partner_info/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Serv. Tec. Avanzados - Pedro M. Baeza (http://www.serviciosbaeza.com)
```

Comment:
@pedrobaeza How should I put it to fit this within 79 characters? Could you please suggest?